src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -449,10 +470,8 @@ private boolean joinEvent(HttpServletRequest request, HttpServletResponse respon
   }
 
   private List<String> IDsToUsernames(List<String> IDs) {
-    return IDs != null
-        ? IDs.stream().map(id -> userStorageObject.getUsernameByID(id))
-            .filter(Objects::nonNull).collect(Collectors.toList())
-        : null;
+    return IDs != null ? IDs.stream().map(id -> userStorageObject.getUsernameByID(id)).filter(Objects::nonNull)
+        .collect(Collectors.toList()) : null;
```

Comment:
That is getting unreadable. Maybe extract the case IDs == null in an if statement.